bryft/src/pages/HomePage/HomePage.js

```diff
@@ -0,0 +1,35 @@
+import "./HomePage.css";
+import React from 'react';
+import Navbar from "../../components/Navbar/Navbar";
+import backdrop from './royce-backdrop.png';
+import icon1 from './icon1.png';
+import icon2 from './icon2.png';
+import icon3 from './icon3.png';
+import { Link } from "react-router-dom";
+
+function HomePage() {
+    return (
+        <React.Fragment>
+            <Navbar />
+            <img className="backdrop" src={backdrop} alt="Backdrop" />
+            <h1>BETTER TRANSPORT FOR BRUINS.</h1>
+            <Link to="/rides"><div className="textbox t1">Create a group</div></Link>
+            <Link to="/rides"><div className="textbox t2">Find a ride</div></Link>
+            <div className="backing"></div>
```

Comment:
For this section, can we try using the css property of "display: flex" to reduce the amount of different css and keep it more concise?
https://css-tricks.com/snippets/css/a-guide-to-flexbox/